www/lendingbot.html

```diff
@@ -37,7 +37,7 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/toastr.min.js"></script>
     <link href="https://cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/toastr.min.css" rel="stylesheet">
 
-    <script src="lendingbot.js?version=1" ></script>
+    <script src="lendingbot.js?version=2" ></script>
```

Comment:
Why do we need this? Ctrl-F5 in browser after upgrade should handle reload without cache. Incrementing it manually after every edit a bit overkill IMHO.